### PR TITLE
Fixing the grammatical errors in the configuration files

### PR DESCRIPTION
**Pull request by Cronus1007**
_describe your changes..._

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [ ] Make sure your PR complies with the [writer's guide][2].
- [ ] Review the diff carefully as sometimes this can reveal issues.
- [ ] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

**Comment by montogeek** [comment]
Please revert `node-sass` change.

**Comment by Cronus1007** [comment]
@montogeek  I have added the Node-Sass Plugin Concept to the documentation.Please review the PR .

File: package.json [hunk 1]
```diff
@@ -120,7 +120,7 @@
     "minimist": "1.2.3",
     "mkdirp": "^0.5.1",
     "modularscale-sass": "^3.0.3",
-    "node-sass": "^4.5.3",
+    "node-sass": "^4.14.1",
```
Comment (snitin315) [inline package.json:123]: unnecessary change, please revert.

File: src/content/plugins/index.md [hunk 2]
```diff
@@ -14,8 +14,35 @@ contributors:
   - chenxsan
 ---
 
-webpack has a rich plugin interface. Most of the features within webpack itself use this plugin interface. This makes webpack __flexible__.
+webpack has a rich plugin interface. Most of the features within webpack itself use this plugin interface. This makes webpack \_ _ flexible_ \_ .
 
+<<<<<<< HEAD
+| Name                                                                              | Description                                                                            |
```
Comment (snitin315) [inline src/content/plugins/index.md:20]: looks like a conflict?

**Review by snitin315 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**